packages/components/src/components/textarea/textarea.tsx

```diff
@@ -175,36 +175,41 @@ export class Textarea {
     return (
       <Host>
         <div class={this.getCssClassMap()}>
-          {/* Accessibility: label should be always *before* the actual input */}
-          <label class="textarea__label" htmlFor={this.inputId}>
-            {this.label}
-          </label>
-          <textarea
-            class="textarea__control"
-            style={!!this.resize && { resize: this.resize }}
-            value={this.value}
-            {...(!!this.name ? { name: this.name } : {})}
-            {...(!!this.inputAutofocus ? { autofocus: 'true' } : {})}
-            required={this.required}
-            minLength={this.minLength}
-            maxLength={this.maxLength}
-            id={this.inputId}
-            onInput={this.handleInput}
-            onChange={this.handleChange}
-            onFocus={this.handleFocus}
-            onBlur={this.handleBlur}
-            onKeyDown={this.handleKeyDown}
-            {...(!!this.placeholder ? { placeholder: this.placeholder } : {})}
-            disabled={this.disabled}
-            {...readonlyAttr}
-            {...(!!this.rows ? { rows: this.rows } : {})}
-            {...(!!this.cols ? { cols: this.cols } : {})}
-            {...ariaInvalidAttr}
-            {...(this.helperText ? ariaDescribedByAttr : {})}
-          />
-
-          {/* Accessibility: solid background for the textarea label to avoid making the label unreadable when there's text underneath */}
-          <span class="textarea__label-safety-background" aria-hidden="true" />
+          <div
+            class="textarea-mask"
```

Comment:
I'm not sure if this follows convention or if should be `textarea__mask` instead?